test/RBdata.sql

```diff
@@ -1498,17 +1498,15 @@ UNLOCK TABLES;
 
 DELETE FROM `ImagingFileTypes`;
 INSERT INTO `ImagingFileTypes` VALUES
-      ('mnc'),
-      ('obj'),
-      ('xfm'),
-      ('xfmmnc'),
-      ('imp'),
-      ('vertstat'),
-      ('xml'),
-      ('txt'),
-      ('nii'),
-      ('nii.gz'),
-      ('nrrd');
+  ('mnc',      'MINC file'),
```

Comment:
Should include the column name in the INSERT INTO statement, this would prevent future break in case of new column

INSERT INTO `ImagingFileTypes` **(type, description)** VALUES ...